build_sysext

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Flatcar Linux. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file. Script to generate sysext. See systemd-sysext(8).
+# Prerequisite that you've run build_packages and build_image
+
+SCRIPT_ROOT=$(dirname "$(readlink -f "$0")")
+. "${SCRIPT_ROOT}/common.sh" || exit 1
+
+# Script must run inside the chroot
+assert_inside_chroot
+assert_root_user
+
+DEFINE_string board "${DEFAULT_BOARD}" \
+  "The board to build an image for."
+
+FLAGS_HELP="USAGE: build_sysext [flags] [package to build].
+This script is used to build a Flatcar sysext. 
+
+Examples:
+
+build_sysext --board=amd64 python - builds python sysext for amd64 architecture.
```

Comment:
```suggestion
sudo build_sysext --board=amd64 python - builds python sysext for amd64 architecture.
```
Can you add a few more and also describe how the resulting file is named and where it will be stored?